schemas/com.apple.swiftui/window_group/jsonschema/1-0-0

```diff
@@ -0,0 +1,47 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "self": {
+    "vendor": "com.apple.swiftui",
+    "name": "window_group",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "description": "Schema for a window group scene entity (VisionOS)",
+  "type": "object",
+  "properties": {
+    "id": {
+      "type": "string",
+      "format": "uuid",
+      "description": "Randomly generated ID for the current window within the group. Generated by the tracker if not provided by the user."
```

Comment:
This can only be generated by the tracker if the tracker then remembers the ID and can automatically attach to all events from the window. But I think this is not currently implemented in the tracker and it's also quite difficult to implemnet because the tracker is not aware which window the event is coming from – there may be multiple open at the same time. So I think we need t o leave up to the users to manage.

```suggestion
      "description": "Randomly generated ID for the current window within the group."
```